src/plugins/intel_cpu/src/nodes/llm_mlp.cpp

```diff
@@ -44,7 +44,7 @@ class LinearKsplit2 {
         bool is_quantized = config.down_quantized;
 
         auto reg_blk_K_size = is_quantized ? REG_BLK_K_SIZE_I8 : REG_BLK_K_SIZE;
-        auto cache_blk_k_size = is_quantized ? CACHE_BLK_K_SIZE : CACHE_BLK_K_SIZE;
+        auto cache_blk_k_size = CACHE_BLK_K_SIZE;
```

Comment:
Looks like the initial code may contain a bug.
@usstq Could you please comment on this?

Reply:
It's not a bug, the change is valid.